### ISSUE TITLE
UI requires stamps so we either need to change the UI or add stamps on the LISP end

### ISSUE THREAD

**Issue by timotree3** [artbrock/cludder#1]

**Comment by matthewjosef** [comment]
Or we can make use of time stamps from when a post is put onto the source chain (header?).

**Comment by artbrock** [comment]
Yeah... the JS version put timestamps into the posts, but since we already timestamp the headers of every entry in the chain, I wanted to try to retrieve those timestamps so we're not making duplicate ones.

I think we can retrieve them from the headers using properties of items put to the DHT.  In any case, we will need that stuff to work, so I'm kind of holding out for it.
